ArduCopter/control_rtl.cpp

```diff
@@ -85,7 +85,7 @@ void Copter::rtl_climb_start()
     wp_nav.wp_and_spline_init();
 
     // RTL_SPEED == 0 means use WPNAV_SPEED
-    if (!is_zero(g.rtl_speed_cms)) {
+    if (!is_zero((float)g.rtl_speed_cms)) {
```

Comment:
I will change it, but i want to add that there have been more of such things in the code.